feature-libs/organization/administration/components/user/form/user-form.component.html

```diff
@@ -99,7 +99,7 @@
       <cx-form-errors [control]="form.get('orgUnit.uid')"></cx-form-errors>
     </label>
 
-    <label *ngIf="isAssignedToApprovers" class="full-width">
+    <fieldset *ngIf="isAssignedToApprovers" class="full-width">
```

Comment:
In case of single input `fieldset` doesn't really make sense. In this case `div` is ok.

Reply:
make sense. Thanks a lot for pointing it out.